PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -301,11 +302,11 @@ extension ArchivedItemsListViewModel {
             selectedFilters.insert(filter)
         }
 
+        fetchLocalItems()
+        
```

Comment:
I moved this up because the snapshot was reporting that there were no items when running lines 307-309. This would make the empty state appear briefly for a quick second before loading the items. Let me know if any issues! Same applies in line 330 in `SavedItemsListViewModel`.